### ISSUE TITLE
When declarative-user-profile feature is enable I can't access any user

### ISSUE THREAD

**Issue by osorionicolas** [keycloak/keycloak#26882]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/api

### Describe the bug

When declarative-user-profile feature is enable and I try to access a user the API /admin/realms/local/users/{USER_ID}?userProfileMetadata=true returns {"error":"unknown_error"} and the UI get stucked until I refresh the page.

### Version

23.0.6

### Expected behavior

It should be possible to access the user information and the UI should not get blocked

### Actual behavior

It shows the following message and you can't do anything else until refresh the page
![image](https://github.com/keycloak/keycloak/assets/26752730/e74455c0-898a-42cb-bf93-f454939e0d8f)

### How to Reproduce?

1. Enable the declarative-user-profile
2. Go to the realm settings tab and enable "User Profile Enabled"
3. Go to the Users tab and try to access any user

**Comment by osorionicolas** [comment]
you are right if I start a fresh instance that error is not happening, I will try to check if there is any configuration in my instance that is generating that behavior

**Comment by pedroigor** [comment]
The user profile is now enabled by default starting from Keycloak 24. The issue is no longer reproducible.
